### ISSUE TITLE
Backup

### ISSUE THREAD

**Issue by heaven2** [mobnetic/BitcoinChecker#195]
I switch ROMs a lot and it would be nice to backup my settings and coins I follow. Right now I manually add everything back.

**Comment by nathankaiser** [comment]
Yes, that would be a great feature. Best would be if identity  would be done through Facebook, Google etc.
